spec/factories/repos/dummy.git/refs/.gitignore

```diff
@@ -0,0 +1 @@
+/remotes/*
```

Comment:
see [here](https://github.com/glittergallery/GlitterGallery/tree/master/spec/factories/repos/dummy.git/refs/remotes) @Alwahsh had added this for test. files like `bare/master` gets created here, after you run test locally. He has added a .gitignore file to ignore everything inside `bare/` but still the folder `bare` would get created and commited if we run test and commit. So we need to ignore everything inside `remote`